### ISSUE TITLE
Investigate if open-jdk EA build failures are legitimate of need to be reported

### ISSUE THREAD

**Issue by nikhilnanivadekar** [eclipse/eclipse-collections#907]
Example log: https://api.travis-ci.org/v3/job/692952556/log.txt

```
18:55:13:770 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project eclipse-collections: Compilation failure: Compilation failure: 
18:55:13:770 [ERROR] /home/travis/build/eclipse/eclipse-collections/eclipse-collections/src/main/java/org/eclipse/collections/impl/string/immutable/CodePointAdapter.java:[56,7] error: types CharSequence and PrimitiveIterable are incompatible;
18:55:13:770 [ERROR]   class CodePointAdapter inherits unrelated defaults for isEmpty() from types CharSequence and PrimitiveIterable
18:55:13:771 [ERROR] /home/travis/build/eclipse/eclipse-collections/eclipse-collections/src/main/java/org/eclipse/collections/impl/string/immutable/CharAdapter.java:[56,7] error: types CharSequence and PrimitiveIterable are incompatible;
18:55:13:772 [ERROR]   class CharAdapter inherits unrelated defaults for isEmpty() from types CharSequence and PrimitiveIterable
18:55:13:772 [ERROR] /home/travis/build/eclipse/eclipse-collections/eclipse-collections/src/main/java/org/eclipse/collections/impl/string/immutable/CodePointList.java:[51,7] error: types CharSequence and PrimitiveIterable are incompatible;
18:55:13:773 [ERROR]   class CodePointList inherits unrelated defaults for isEmpty() from types CharSequence and PrimitiveIterable
18:55:13:774 [ERROR] -> [Help 1]

```

**Comment by nikhilnanivadekar** [comment]
I am working on this. Issue raised only for tracking.

**Comment by nikhilnanivadekar** [comment]
This is fixed in 10.4.0
